examples/tv-app/android/java/MyUserPrompterResolver-JNI.cpp

```diff
@@ -64,3 +64,38 @@ JNI_METHOD(void, OnPromptDeclined)(JNIEnv *, jobject)
     GetCommissionerDiscoveryController()->Cancel();
 #endif
 }
+
+JNI_METHOD(void, OnCommissionerPasscodeOK)(JNIEnv *, jobject)
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_BOTH_COMMISSIONER_AND_COMMISSIONEE
+    chip::DeviceLayer::StackLock lock;
+    ChipLogProgress(Zcl, "OnCommissionerPasscodeOK");
+    // GetCommissionerDiscoveryController()->Ok();
```

Comment:
Commented out by mistake?

Reply:
there is no callback right now to its more of a TODO